### PR TITLE
Send to Note + Multi-Note + Multi-Window

### LINKED ISSUE #787: Send to Note + Multi-Note + Multi-Window
### [Description]:

- [x] Make sure "Send to Note" can work with multiple windows open (sidebars)
 If you have 3 windows it will create 3 notes

Extracted from: https://github.com/mozilla/notes/pull/743

### PR DESCRIPTION
**Pull request by sebastienbarbier**
On send-to-note, we propagate windowId from background to react app which perform the action only if nto from an other window.
Have to be performed in sidebar react app since background do not know which note is displayed.

Fix #787

### CODE REVIEW

**Comment by vladikoff** [comment]
@sebastienbarbier if I have two windows open and I send to notes then one window creates a new note, the other window adds contents of it to an opened note:

**Comment by sebastienbarbier** [comment]
I couldn't reproduce and will ask the dummy question , are you sure you were in `787-send-to-notes-multi-window` branch ? (yep, sorry for that, I'm that desperate 😅)

**Comment by vladikoff** [comment]
@sebastienbarbier hm maybe it didn't rebuild, sorry, let me try again

**Comment by vladikoff** [comment]
@sebastienbarbier cannot trigger it even with 3 windows, seems to work! sorry !
